partner_communication_switzerland/models/contracts.py

```diff
@@ -464,3 +465,16 @@ def _new_dossier(self):
             if sub.correspondant_id.id != sub.partner_id.id:
                 sub.send_communication(
                     sub_proposal_config, correspondent=False)
+
+
+class ContractGroup(models.Model):
+    _inherit = 'recurring.contract'
+
+    send_to_4m = fields.Date('Sent to 4M')
```

Comment:
The class above is already the recurring.contract class, so you can add it above instead of doing a new class.

Reply:
The name shoud be `sent_to_4m`